internal/http/services/owncloud/ocdav/put.go

```diff
@@ -184,10 +191,24 @@ func (s *svc) handlePut(w http.ResponseWriter, r *http.Request, ns string) {
 		}
 	}
 
+	length, err := strconv.ParseInt(r.Header.Get("Content-Length"), 10, 64)
```

Comment:
Check that we have valid length?

Reply:
https://github.com/cs3org/reva/issues/702